contracts/test/DummyCollateralizedContract.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity 0.4.18;
+
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+
+import "../examples/Collateralized.sol";
+
+contract DummyCollateralizedContract is Collateralized {
+    using SafeMath for uint;
+
+    function DummyCollateralizedContract(address _debtRegistry) Collateralized(_debtRegistry) public {}
+
+    function registerRepayment(
+        bytes32 agreementId,
+        address payer,
+        address beneficiary,
+        uint256 unitsOfRepayment,
+        address tokenAddress
+    ) public returns (bool _success) {
+      return true;
+    }
+
+    function getExpectedRepaymentValue(
```

Comment:
You will want to add some sort of ability to programmatically change the returned `expectedRepaymentValue` and `valueRepaid` -- these values affect logic of whether a collateralized asset can be seized or not, and you'll want to test for that.

Reply:
Yup. This is just scaffolding. That functionality will come in a future PR.